bears/coffee_script/CoffeeLintBear.py

```diff
@@ -21,20 +20,28 @@ def convert_if_not_empty(value: str, conversion):
     return None
 
 
-class CoffeeLintBear(LocalBear, Lint):
-    executable = 'coffeelint'
-    arguments = '--reporter=csv {filename}'
+@linter(executable='coffeelint')
+class CoffeeLintBear:
+    """
+    Coffeelint's your files!
```

Comment:
@Makman2 don't advertise CoffeeLint, advertise your bear

The first line must be a brief description of what it can do not what it wraps. Nobody cares what it wraps! They use coala to not use all those tools, so don't make them look them up if they don't need to.